asv/environment.py

```diff
@@ -416,7 +416,9 @@ def check_presence(self):
 
         for executable in ['pip', 'python']:
             exe = self.find_executable(executable)
-            if not os.path.isfile(exe):
+            # do not count a system install of pip or python
```

Comment:
Empty line before comment blocks, when at the same intendation level as the preceding line.